client/my-sites/site-settings/disconnect-site/another-reason.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormTextArea from 'components/forms/form-textarea';
+import SectionHeader from 'components/section-header';
+import { addQueryArgs } from 'lib/url';
+
+class AnotherReason extends PureComponent {
+	state = {
+		reason: '',
+	};
+
+	handleChange = event => {
+		this.setState( { reason: event.target.value } );
+	};
+
+	render() {
+		const { confirmHref, translate } = this.props;
+
+		return (
+			<div className="disconnect-site__another-reason">
+				<SectionHeader
+					label={ translate(
+						'Please share why you want to disconnect your site from WordPress.com.'
+					) }
+				/>
+				<Card>
+					<FormTextArea onChange={ this.handleChange } value={ this.state.reason } />
```

Comment:
Looking at it, I'm wondering whether it would be easier to analyze results if we're using a `FormInput` instead of `FormTextarea`. I'd assume that shorter answers might be easier to analyze. Also, since this value goes into the URL, it might result in an ugly URL, or even in an invalid URL if the user tries to input a bunch of text there.

Reply:
From the data perspective we do hope for keywords 🤞 :)